### PR TITLE
feat: Add php support for anonymous attribute redaction in feature events

### PR DESCRIPTION
**Pull request by keelerm84**

### CODE REVIEW

File: sdktests/common_tests_events_contexts.go [hunk 1]
```diff
@@ -51,7 +51,6 @@ func makeEventContextTestParams() []eventContextTestParams {
 					b.Kind("org")
 					b.Name("a")
 					b.SetString("b", "c")
-					b.Anonymous(true)
```
Comment (cwaldren-ld) [inline sdktests/common_tests_events_contexts.go:54]: Why this change?

File: sdktests/php_events_eval.go [hunk 2]
```diff
@@ -150,6 +150,123 @@ func doPHPFeatureEventTests(t *ldtest.T) {
...
+
+			for _, valueType := range getValueTypesToTest(t) {
+				t.Run(testDescFromType(valueType), func(t *ldtest.T) {
+					flag := flagFactories[flagSelectors{
+						tracked:   true,
+						withDebug: false,
+						malformed: false,
+					}].ReuseFlagForValueType(valueType)
+					var expectedValue ldvalue.Value
+					// var expectedVariation o.Maybe[int]
```
Comment (cwaldren-ld) [inline sdktests/php_events_eval.go:170]: commented code

File: sdktests/common_tests_events_contexts.go (hunk 1 above)
Comment (keelerm84) [reply sdktests/common_tests_events_contexts.go:54]: These tests are used for PHP and non-PHP server side SDKs.

If the SDK is PHP, it takes a branch that tries to validate the context provided in that event. You can change what that expectation looks like but it also affects the index event.

Non-PHP doesn't have this same branch. By removing the anonymous flag, the feature event context won't be different from the index one, and I don't have to change a ton of stuff.

Also, I have a test verifying the anonymous behavior of PHP with feature events, so no coverage is lost.

File: sdktests/php_events_eval.go (hunk 2 above)
Comment (keelerm84) [reply sdktests/php_events_eval.go]: Done!

**Review by cwaldren-ld — APPROVED** [review]
